lib/view/page/ideas_main_page.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+import 'package:idea_tracker/view/page/create_idea_page.dart';
+import 'package:idea_tracker/view/page/idea_details_page.dart';
+
+class IdeasMainPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
```

Comment:
Don't wrap pages with MaterialApp. If you want to change the Theme, wrap with Theme.